### PR TITLE
Adding Jira-Project-Config and In Progress Config

### PR DESCRIPTION
**Pull request by CanardSauvage**

### CODE REVIEW

File: dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jira/JiraProjectConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import de.axelspringer.ideas.tools.dash.business.jira.rest.Issue;
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class JiraProjectConfiguration {
+
+    private final List<String> issueStatesInProgress = new ArrayList<>();
```
Comment (gaffa) [inline dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jira/JiraProjectConfiguration.java]: is there a chance we provide something like a more or less sane default?

Comment (CanardSauvage) [inline dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jira/JiraProjectConfiguration.java]: Yes, the sane default would be "In Progress" and would work for the other known dashes out of the box. But I thought "no highlighting" is a better default.

**Comment by CanardSauvage** [comment]
@wThomas84 There are a lot of parameters for the jira-checks, if we make one configurable with properties we would need to make everything configurable with properties. And properties would increase the complexity, as the configuration would be farther away from the CheckProvider one would need to implement nevertheless.

**Comment by gaffa** [comment]
Property-files are less straightforward, I agree. I decided to merge the PR to get things forward. If we want defaults we can suggest a new PR for those.

**Pull request merged**